docs/data-sources/equinix_metal_devices.md

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Metal"
+---
+
+# equinix_metal_devices
+
+Provides an Equinix Metal devices datasource. This can be used to find devices that meet a filter criteria.
```

Comment:
Perhaps we should mention that `equinix_metal_device` is also available for fetching a single device.  Likewise, perhaps we should add the reverse note in `equinix_metal_device` docs.